tobac/feature_detection.py

```diff
@@ -1005,6 +1009,14 @@ def feature_detection_multithreshold_timestep(
     # get actual numpy array and make a copy so as not to change the data in the iris cube
     track_data = data_i.core_data().copy()
 
+    # keep a copy of the unsmoothed data (that can be used for calculating stats)
```

Comment:
Do we need to copy the data here, or can we just pass `data_i` directly to `get_statistics` if calculating on the unsmoothed data?

Reply:
No, you are right. I changed this to directly pass `data_i` to `get_statistics()`. Thanks!